pom.xml

```diff
@@ -151,8 +151,8 @@
         <artifactId>maven-compiler-plugin</artifactId>
         <version>3.2</version>
         <configuration>
-          <source>1.7</source>
-          <target>1.7</target>
+          <source>9</source>
+          <target>9</target>
```

Comment:
Java 8 is still widely used.  Can we keep the source/target 8?

Reply:
The development require >Java9 , but it does NOT mean need to change compile source/target level to 9. I have do local dev for Java11, have not seen any issue.
Could you give one example of APIs required Java9?

Reply:
> Would changing target to 9 cause breaking changes?
It will need to <JRE9 can not use javassist anymore.